src/System.Net.Primitives/tests/FunctionalTests/System.Net.Primitives.Functional.Tests.csproj

```diff
@@ -42,6 +43,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <!-- TODO: Remove this reference once packages are updated -->
```

Comment:
@stephentoub If this intermediate hop is needed, shouldn't this reference the src/csproj instead? 
Also, I thought that packages are used from the local build: why isn't it working now that you've added the implementation to Primitives?